src/Admin/AbstractAdmin.php

```diff
@@ -1487,6 +1489,18 @@ public function attachAdminClass(FieldDescriptionInterface $fieldDescription)
 
             if (!$pool->hasAdminByClass($targetModel)) {
                 return;
+                // NEXT_MAJOR: Uncomment the following exception instead.
+//                throw new \InvalidArgumentException(sprintf(
+//                    'No admin found for the class "%s", please use the admin_code option instead',
+//                    $targetModel
+//                ));
+            }
+
+            if (!$pool->hasSingleAdminByClass($targetModel)) {
+                throw new \InvalidArgumentException(sprintf(
+                    'Too many admins found for the class "%s", please use the admin_code option instead',
```

Comment:
```suggestion
                    <<<'EXCEPTION'
Context: trying to find an admin for class "%s"
Problem: That class has more than one admin.
Solution: Use the unambiguous admin_code option instead.
EXCEPTION,
```

Reply:
Won't it lead to weird display on 7.2 ?

I would prefer to do this change in a more general way on our code ? Maybe even with an helper to generate an exception message ?
```
public static function formatExceptionMessage($context, $problem, $solution)
```